### DISCUSSION TITLE
Implement spawnProcess option within window.create API

### DISCUSSION THREAD

**Discussion by Blatman** [neutralinojs/neutralinojs#918] [Ideas]
At the moment the window.create API uses the execCommand API and returns the PID which can be used for window management. Just thinking that the new spawnProcess API gives the option of process update and exit features so would this be useful to enhance the window create process? Closing of the window could use exit with the spawnProcesses id and maybe the updateSpawnProcess could be used to refresh a windows contents from the main app.

**Comment by shalithasuranga** [comment]
Hey.. @Blatman it's a great idea, Thanks :tada: we can try to release this with an upcoming release :)
